source/_patterns/02-molecules/navigation/breadcrumbs/breadcrumbs.yml

```diff
@@ -1,6 +1,10 @@
-breadcrumbs:
-  - url: "#"
-    label: "Debt Collection"
-    url_attributes: ""
-  - label: "Pay in Installments"
-    url_attributes: ""
+# Font styling variants ... font_variant:
+#  - trialcourt
+
+breadcrumb:
```

Comment:
@melwong-jcc 
adding this yml root will require a change in how breadcrumbs are referenced, such as in hero:
https://github.com/Exygy/courtyard/blob/59286be8dc1f995c5d11b431633161b8c426fee0/source/_patterns/02-molecules/blocks/hero/hero.twig#L26

and wherever hero.twig is referenced in patternlab and SRL:
https://github.com/chapter-three/jcc-srl/search?q=hero.twig&unscoped_q=hero.twig

Breadcrumbs are appearing fine because the defaults are getting loaded.  The values passed by hero is not taking effect.

Reply:
Oh right. Missed that. This might explain why Switchboard appeared not working because there were no values in switchboard.yml. Thanks for catching the missed adjustments needed. Luckily the breadcrumb component is only used is a few places.

Reply:
Can you review the latest? Impacted components, templates, and pages now adjusted except for new forms lookup which is outdated and Benny is okay not updating.